health/micro-ui/web/micro-ui-internals/packages/modules/health-payments/src/components/editAttendeesPopUp.js

```
@@ -0,0 +1,237 @@
+        setSearchUserpopUp(false);
+    };
+
+
+    // -------- Render --------
+    return (<React.Fragment>
+        <PopUp
+            style={{ minWidth: "1000px" }}
+            onClose={onClose}
+            heading={t(heading)}
+            onOverlayClick={onClose}
+            children={[
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)